VFighter/Assets/Scripts/GravityObjectRigidBody.cs

```diff
@@ -146,7 +163,28 @@ public void ChangeGravityScale(float newGravityScale)
 
     void OnCollisionEnter2D(Collision2D collision)
     {
-        if(_stopObjectOnCollide)
+        if(collision.gameObject.GetComponent<GravityObjectRigidBody>())
+        {
+            var vel = _rB.velocity;
+            var normal = Vector2.zero;
+            foreach (var contact in collision.contacts)
+            {
+                normal += contact.normal;
+                break;
+            }
+
```

Comment:
Remove this space

Reply:
so after a ending brace always leave an empty line